src/gui/topbar_subwindows/politics_subwindows/gui_reforms_window.hpp

```diff
@@ -141,6 +141,12 @@ void describe_reform(sys::state& state, text::columnar_layout& contents, dcon::i
 		if((rules & issue_rule::build_railway) != 0) {
 			text::add_line(state, contents, "rule_build_railway");
 		}
+		if((rules & issue_rule::build_bank) != 0) {
+			text::add_line(state, contents, "rule_build_bank");
```

Comment:
because you made this rule == building railroads, this text will show up even when banks aren't enabled (so just wrap the display in a check for the building being enabled)